client/landing/stepper/hooks/use-is-site-big-sky-eligible.ts

```diff
@@ -33,6 +33,12 @@ export function useIsBigSkyEligible() {
 	const isEligibleGoals = isGoalsBigSkyEligible( goals );
 	const isEligiblePlan = isPremiumPlan( product_slug ) || isBusinessPlan( product_slug );
 
+	if ( config.isEnabled( 'onboarding/big-sky-before-plans' ) ) {
+		const eligibilityResult =
+			( featureFlagEnabled && isEligibleGoals && onSupportedDevice ) || false;
```

Comment:
I used a separate branch for this logic because we also need to exclude the `isOwner` logic

Reply:
Sounds good, I just [removed this || false](https://github.com/Automattic/wp-calypso/pull/99019/commits/9698ac067ed54fb8bc158a0984c18c7a4b551c55) logic from this file as we don't need it.